sdk/JavaScript/TestFramework/js/testFramework.js

```diff
@@ -36,15 +36,15 @@ function createZumoNamespace() {
         this.canRunUnattended = true;
         this.logs = [];
         this.requiredFeatures = [];
-        if (requiredRuntimeFeatures) {
-            if (typeof requiredRuntimeFeatures === 'string') {
-                requiredRuntimeFeatures = [requiredRuntimeFeatures];
+        if (requiredRuntimeFeatures) {
```

Comment:
Sublime does all tabs to spaces for me, yeah.  You're right, it's impossible to see the changes.